src/shaders/brc/skl/Makefile.am

```diff
@@ -0,0 +1,17 @@
+INTEL_BRC_SKL =			 	\
+	hme_downscale_genx_0.g9b 	\
+	hme_genx_0.g9b			\
+	vp8_brc_init_genx_0.g9b		\
+	vp8_brc_reset_genx_0.g9b	\
+	vp8_brc_update_genx_0.g9b	\
+	vp8_enc_genx_0.g9b		\
+	vp8_enc_genx_1.g9b		\
+	vp8_enc_genx_2.g9b		\
+	vp8_intra_distortion_genx_0.g9b \
+	vp8_mpu_genx_0.g9b		\
+	vp8_tpu_genx_0.g9b
+
```

Comment:
Same concern on adding a LICENSE file with the APACHE 2 text to the same folder.